test/metabase/api/dashboard_test.clj

```diff
@@ -2829,7 +2829,19 @@
                   :has_more_values false}
                  (->> (chain-filter-values-url (:id dashboard) (:category-name param-keys))
                       (mt/user-http-request :rasta :get 200)
-                      (chain-filter-test/take-n-values 3)))))))))
+                      (chain-filter-test/take-n-values 3)))))))
+
+    (testing "block data perms should not allow access to field values"
```

Comment:
This `deftest` is already way too big, I really think you should just make this its own `deftest`. Otherwise debugging this is going to be a nightmare since you'll have to wade thru like 10 other separate test cases